### PR TITLE
New version: LyX.LyX version 2.3.7.0

### LINKED ISSUE #145500: [Update Request]: LyX
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

LyX.LyX

### Package Version

2.3.7

### Please describe the changes you would like to see

There is a new release available (2.3.7). Download page is: [https://www.lyx.org/Download](url)

### PR DESCRIPTION
**Pull request by schembriaiden**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves: https://github.com/microsoft/winget-pkgs/issues/145500

---

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 870.)

**Comment by vikingnope** [comment]
@stephengillie, any ideas of what can be done here?

**Comment by vikingnope** [comment]
It does say this on the website next to the download section "NOTE: Before you install LyX on Windows, you need to install a TeX distribution. For more information on how to do this, please see [this page](https://wiki.lyx.org/Windows/TeXLive) on the LyX wiki."

File: manifests/l/LyX/LyX/2.3.7.0/LyX.LyX.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: LyX.LyX
+PackageVersion: 2.3.7.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
```
Comment (stephengillie) [inline manifests/l/LyX/LyX/2.3.7.0/LyX.LyX.installer.yaml:12]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: MiKTeX.MiKTeX
Installers:
```

(Automated response - build 872.)

**Comment by stephengillie** [comment]
I tried with TeXstudio as a dependency, which doesn't install LaTeX, but also depends on it. It hinted to use MiKTeX. Unfortunately, this didn't let the install complete, only changed errors:

image

**Comment by vikingnope** [comment]
Did you try with TeXLive?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-03-22 09:13:43.390 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFAC2A4C11F: (caller: 00007FFAC27D8D6F) LogHr(1) tid(16e8) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFAC2952B08: (caller: 00007FFAC29FBC20) Exception(1) tid(16e8) 8A15005E ] 
> 2024-03-22 09:13:43.390 [REPO] Failed to open available source: msstore
> 2024-03-22 09:13:51.201 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFAA445C11F: (caller: 00007FFAA41E8D6F) LogHr(1) tid(15cc) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFAA4362B08: (caller: 00007FFAA440BC20) Exception(1) tid(15cc) 8A15005E ] 
> 2024-03-22 09:13:51.201 [REPO] Failed to open available source: msstore
> 2024-03-22 09:13:15.530 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!6974535B: (caller: 69641A80) LogHr(1) tid(1df0) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!6978B5DD: (caller: 698212BF) Exception(1) tid(1df0) 8A15005E ] 
> 2024-03-22 09:13:15.530 [REPO] Failed to open available source: msstore
> 2024-03-22 09:13:22.985 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!548C535B: (caller: 547C1A80) LogHr(1) tid(4b0) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!5490B5DD: (caller: 549A12BF) Exception(1) tid(4b0) 8A15005E ] 
> 2024-03-22 09:13:22.985 [REPO] Failed to open available source: msstore
> 2024-03-22 09:13:24.182 [CLI ] No suitable installer found for manifest MiKTeX.MiKTeX with version 24.1

(Automated response - build 873.)

**Comment by stephengillie** [comment]
> 2024-03-22 09:13:24.182 [CLI ] No suitable installer found for manifest MiKTeX.MiKTeX with version 24.1

I believe this is from the `x86` install, as our `x86` validations are performed on 32-bit OSes, and `MiKTeX.MiKTeX` only has an `x64` version.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-06 05:34:47.641 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFC8D99C11F: (caller: 00007FFC8D728D6F) LogHr(1) tid(1b94) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC8D8A2B08: (caller: 00007FFC8D94BC20) Exception(1) tid(1b94) 8A15005E ] 
> 2024-04-06 05:34:47.641 [REPO] Failed to open available source: msstore
> 2024-04-06 05:34:56.234 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFC76E2C11F: (caller: 00007FFC76BB8D6F) LogHr(1) tid(1534) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC76D32B08: (caller: 00007FFC76DDBC20) Exception(1) tid(1534) 8A15005E ] 
> 2024-04-06 05:34:56.234 [REPO] Failed to open available source: msstore

(Automated response - build 876.)

**Comment by vikingnope** [comment]
might have to forget about this package then

**Pull request closed without merging**